sorc/machine-setup.sh

```diff
@@ -210,6 +188,9 @@ fi
 # Source versions file for build
 
 . ${HOMEgfs}/versions/build.ver
+if [[ $target != wcoss2 ]] ; then
```

Comment:
I'd prefer to have a blank wcoss2.ver file and not have this if-block

Reply:
Created `wcoss2.ver` and updated `machine-setup.sh` @ 1901756